docs/demo-environment.md

```diff
@@ -0,0 +1,13 @@
+---
+id: demo-environment
+title: Demo environment
+sidebar_position: 5
+---
+
+# Demo environment
+
+To make it easier to try out Fuse, we provide a demo environment.
+
+To use it, specify the `--with-demo-env` flag. By default, when running [`fuse scenarios`](./scenarios.md), you will be prompted if you want to use it as a part of the guided paths.
```

Comment:
```suggestion
To use it, specify the `--with-demo-env` flag. By default, when running [`fuse tutorials`](./tutorials.md), you will be prompted if you want to use it as a part of the guided paths.
```

based on your previous suggestion, we prefer the `tutorials` command, while `scenarios` is only an alias, this needs to be applied also for `scenarios.md` file.